From 5d2271ca9f5f569bbec950f9f0cc98af070ea52f Mon Sep 17 00:00:00 2001
From: Matt Falkenhagen <falken@chromium.org>
Date: Wed, 17 Oct 2018 10:12:44 +0000
Subject: [PATCH] Enable ServiceWorkerServicification by default.

Bug: 715640, 846235
Change-Id: I49ed8a5a3bc5962cd52e24ab734b00fdfdd7b32d
Reviewed-on: https://chromium-review.googlesource.com/c/1286238
Commit-Queue: Matt Falkenhagen <falken@chromium.org>
Reviewed-by: Kinuko Yasuda <kinuko@chromium.org>
Cr-Commit-Position: refs/heads/master@{#600337}
---
 third_party/blink/common/features.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third_party/blink/common/features.cc b/third_party/blink/common/features.cc
index a5cf355e1b..b58abdb515 100644
--- a/third_party/blink/common/features.cc
+++ b/third_party/blink/common/features.cc
@@ -59,7 +59,7 @@ const base::Feature kServiceWorkerParallelSideDataReading{
 // Enable new service worker glue for NetworkService. Can be
 // enabled independently of NetworkService.
 const base::Feature kServiceWorkerServicification{
-    "ServiceWorkerServicification", base::FEATURE_DISABLED_BY_DEFAULT};
+    "ServiceWorkerServicification", base::FEATURE_ENABLED_BY_DEFAULT};
 
 // Freeze scheduler task queues in background after allowed grace time.
 // "stop" is a legacy name.
